lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -27,6 +27,63 @@ class WP_Font_Library {
 		'woff2' => 'font/woff2',
 	);
 
+	/**
+	 * Font collections.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var array
+	 */
+	public static $collections = array();
```

Comment:
No need for this to be static.

Reply:
I respectfully disagree. Reasoning is shared here https://github.com/WordPress/gutenberg/pull/53816/files#r1300561853.